rotkehlchen/types.py

```diff
@@ -706,6 +706,7 @@ class Location(DBCharEnumMixIn):
     POLYGON_POS = 40  # on-chain Polygon POS events
     ARBITRUM_ONE = 41  # on-chain Arbitrum One events
     BASE = 42  # on-chain Base events
+    WOO = 43
```

Comment:
don't forget to rebase your PR (gnosis got 43 location) and insert it in the db upgrade (they are a list also in the db)